icesat2_boreal_stac/constants.py

```diff
@@ -51,3 +47,264 @@ def matches_file(self, filename: str) -> bool:
+product, to predict 30-m AGBD per 90-km tile. Estimates of mean AGBD and mean
+vegetation height with standard deviation are provided in cloud-optimized
+GeoTIFF (CoG) format. The product consists of a set of raster grids and tabular
+(CSV) files referenced to a set of 90-km tiles that cover the circumpolar boreal
+domain and south to 51.6°N (Figure 1). Each raster grid is a 2-band file where
+the first and second band represent the mean and standard deviation pixel values
+that result from the bootstrapped prediction. The CSV files feature the ICESat-2
+ATL08 30 m segment centroids that were used as training data in the prediction
+of each raster. A polygon map of these data tiles is included as a GeoPackage
+file and a Shapefile. This product was generated on the NASA-ESA Multi-Mission
+Algorithm and Analysis Platform (MAAP, https://scimaap.net), an open science
+platform. All code and input files are publicly available:
+[https://github.com/lauraduncanson/icesat2_boreal.git](https://repo.ops.maap-project.org/icesat2_boreal/icesat2_boreal.git).
+
+For each product (AGB and height) there are 3902 cloud-optimized GeoTIFFs, 3902 tables
+in comma-separated values (CSV) format, and 1 geopackage tile index."""
+
+COLLECTION_CITATION = """Duncanson, Montesano, Neuenschwander, Zarringhalam, Minor,
+Thomas. Circumpolar boreal aboveground biomass mapping with ICESat-2. (in prep.)"""
+
+PROVIDERS = [
+    Provider(
+        name="MAAP",
+        description="The MAAP platform is designed to combine data, algorithms, and "
+        "computational abilities for the processing and sharing of data related to "
+        "NASA’s GEDI, ESA’s BIOMASS, and NASA/ISRO’s NISAR missions",
+        url="https://maap-project.org",
+        roles=[ProviderRole.PROCESSOR, ProviderRole.PRODUCER],
+        extra_fields={"processing:level": "L4"},
+    )
+]
+
+SUMMARIES = Summaries(
+    summaries={
+        "platform": [
+            # list out platforms included in HLS
+            "LANDSAT-8",
+            "LANDSAT-9",
+            "SENTINEL-2A",
+            "SENTINEL-2B",
+            # others
+            "ICESat-2",
+        ],
+        "instruments": [
+            "Advanced Topographic Laser Altimeter System",
+            "Operational Land Imager",
+            "Operational Land Imager 2",
+            "Sentinel-2 Multispectral Imager",
+        ],
+        "mission": [
+            "ABoVE",
+        ],
+        "gsd": {
+            "minimum": 30,
+            "maximum": 30,
+        },
+    }
+)
+
+KEYWORDS = ["BIOMASS", "VEGETATION HEIGHT"]
+
+COLLECTION_TITLES = {
+    Variable.AGB: "Icesat2 Boreal v2.1: Gridded Aboveground Biomass Density",
+    Variable.HT: "Icesat2 Boreal v2.1: Vegetation Height",
+}
+
+COLLECTION_ASSETS = {
+    Variable.AGB: {
+        "tiles": Asset(
+            href="s3://maap-ops-workspace/shared/montesano/databank/daac/boreal_tiles_v004_AGB_H30_2020_ORNLDAAC.gpkg",
```

Comment:
@hrodmn if this is part of the official data, shouldn't we put a copy in the nasa-maap-data-store?

Reply:
Yes we should. I'll make a ticket to move it over then will update this ref.